shakenfist/virt.py

```diff
@@ -103,7 +109,7 @@ def _populate_block_devices(self):
             'devices': [
                 {
                     'type': disk_type,
-                    'size': self.db_entry['disk_spec'][0].get('size'),
+                    'size': _safe_int_cast(self.db_entry['disk_spec'][0].get('size')),
```

Comment:
If it's not an integer then what is the value?
At this point, the size has to be an integer?
A `.get('size', 0)` could be neater but would push the exception/error further down the road?

Reply:
To unblock your CI work lets accept this, and promise this debt will be repaid during the API input clean up.

Reply:
Yeah, I was thinking about how get() can return None, but you're right that providing a default would work. That said this is still wrong. We use None later as a special case for "detect", so I need to fix this.

Reply:
No wait its right. We want int or None here.